README.md

```diff
@@ -194,6 +194,19 @@ Removes url protocol
 withoutProtocol('http://example.com')
 ```
 
+### `traversePathTree`
+
+Steps through an url removing the last segment each time, giving you a tree of urls.
```

Comment:
Mentioning real-world use case example (as you mentioned breadcrumb) would be nice 👍🏼